docs/source/type_info.rst

```diff
@@ -26,13 +26,18 @@ bits        int     The number of bits occupied by the type.
 eps         float   The smallest representable number such that ``1.0 + eps != 1.0``.
 max         float   The largest representable number.
 min         float   The smallest representable number (typically ``-max``).
-tiny        float   The smallest positive representable number.
+tiny        float   The smallest positive normal number. See notes.
 resolution  float   The approximate decimal resolution of this type, i.e., ``10**-precision``.
 ==========  =====   ========================================
 
 .. note::
   The constructor of :class:`torch.finfo` can be called without argument, in which case the class is created for the pytorch default dtype (as returned by :func:`torch.get_default_dtype`).
 
+.. note::
+  `tiny` returns the smallest *normal* number, but there are smaller
+  subnormal numbers. See https://en.wikipedia.org/wiki/Denormal_number
```

Comment:
"denormal numbers" would be consistent with the Wiki link

Reply:
Yes but the floating point spec is what we truly want to be consistent with and it uses subnormal.

I can change the wiki link to https://en.wikipedia.org/wiki/Subnormal_number which redirects to the Denormal_number page. WDYT?

Reply:
Probably not worth bothering